src/room/useLoadGroupCall.ts

```diff
@@ -80,7 +80,7 @@ export const useLoadGroupCall = (
       const room = await fetchOrCreateRoom();
       logger.debug(`Fetched / joined room ${roomIdOrAlias}`);
       const groupCall = client.getGroupCallForRoom(room.roomId);
-      logger.debug("Got group call", groupCall);
+      logger.debug("Got group call", groupCall.groupCallId);
```

Comment:
`groupCall` can be null, which would make this explode

Reply:
good point